client/components/DateSelector/options.js

```diff
@@ -0,0 +1,64 @@
+import { DateUtils } from 'react-day-picker';
+
+const MMDD = 'MMDD';
+const MMYYYY = 'MMYYYY';
+
+const today = new Date();
+
+const oneMonthBack = DateUtils.addMonths(new Date(), -1);
+const sixMonthsBack = DateUtils.addMonths(new Date(), -6);
+const twelveMonthsBack = DateUtils.addMonths(new Date(), -12);
+const startOfThisYear = new Date(`1/1/${today.getFullYear()}`);
+const oneWeekBack = new Date(new Date().setDate(today.getDate() - 7));
+
+const convertDatesToFormattedString = (dates, format) => {
+  const localeDateString = dates
+    .map(date => date.toLocaleDateString('en-Us').split('/'))
+    .map(dateArr => {
+      const [day, month, year] = dateArr;
+      if (format === MMYYYY) return [month, year].join('/');
+      if (format === MMDD) return [month, day].join('/');
+    });
+
+  return localeDateString.join(' - ');
+};
+
+const options = [
+  {
+    text: `Last Week ( ${convertDatesToFormattedString(
+      [oneWeekBack, today],
+      MMDD,
+    )})`,
+    dates: [oneWeekBack, today],
+  },
+  {
+    text: `Last Month (${convertDatesToFormattedString(
+      [oneMonthBack, today],
+      MMYYYY,
+    )})`,
+    dates: [oneMonthBack, today],
+  },
+  {
+    text: `Last 6 months (${convertDatesToFormattedString(
+      [sixMonthsBack, today],
+      MMYYYY,
```

Comment:
and MMDD here

Reply:
What about the last six months spans back to the last year? For example from January 2021 to July 2020.

Reply:
Yeah, considered that as well. I figured for any timespan less than 1 year, it's useful and more informative for the user to show DD in the displayed date range. What do you think?

Reply:
Per team discussion, remove displayed date ranges from dropdown.